search/tests/test_sorting.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
This whole file just tests some constants and isn't necessary. It doesn't improve test coverage except for a single `if` statement, `search_sorting.py` line 42. It would be better to add a call to `get_sorting_option` with a missing field and delete this file.